lib/components/regions_map/regions_map.dart

```diff
@@ -5,12 +5,16 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:seeds/components/regions_map/components/regions_serach_bar.dart';
+import 'package:seeds/components/regions_map/interactor/view_models/page_commands.dart';
+import 'package:seeds/components/regions_map/interactor/view_models/place.dart';
 import 'package:seeds/components/regions_map/interactor/view_models/regions_map_bloc.dart';
 import 'package:seeds/design/app_colors.dart';
 import 'package:seeds/domain-shared/page_state.dart';
 
 class RegionsMap extends StatefulWidget {
```

Comment:
do we need this to be stateful?

Reply:
I think so since we use a map controller, also the example from this package usea a stateful.